### ISSUE TITLE
Possible Improvement?

### ISSUE THREAD

**Issue by gishmel** [ctindel/reader#2]
Is it possible due to the fact that not all applications should use Stormpath that you create an example case of how to properly handle your own users in MongoDB and Mongoose? I find this to be one of the most lacking examples and its a shame because I have used Stormpath for toy applications and they are good for exactly that but for full featured applications I don't know of anyone who would want to use it. It would be helpful to the community in my opinion to see Users handled in the applications code itself.

**Comment by ctindel** [comment]
Yeah, maybe I could move to doing something like this eventually.  Sure would make it nice when I'm coding on a plane.  :)

https://scotch.io/tutorials/easy-node-authentication-setup-and-local
